xarray/tests/test_dataarray.py

```diff
@@ -6561,47 +6584,56 @@ def test_rolling_properties(da):
         da.rolling(time=2, min_periods=0)
 
 
+@requires_bottleneck
 @pytest.mark.parametrize("name", ("sum", "mean", "std", "min", "max", "median"))
-@pytest.mark.parametrize("center", (True, False, None))
 @pytest.mark.parametrize("min_periods", (1, None))
 @pytest.mark.parametrize("backend", ["numpy"], indirect=True)
-def test_rolling_wrapped_bottleneck(da, name, center, min_periods):
-    bn = pytest.importorskip("bottleneck", minversion="1.1")
+@pytest.mark.parametrize("center", (True, False, None))
```

Comment:
failing for `center=True` or `pad=False` (I think).

I think there's an issue with constructing `expected`: it doesn't seem like `get_expected_rolling_indices` is correct for all cases.

Reply:
@cmdupuis3 I fixed the merge conflicts but there's still failing tests. We'd appreciate any help you can give here.